src/main/java/com/cryptomorin/xseries/XItemStack.java

```diff
@@ -88,28 +91,39 @@ public static void serialize(@Nonnull ItemStack item, @Nonnull ConfigurationSect
         Objects.requireNonNull(item, "Cannot serialize a null item");
         Objects.requireNonNull(config, "Cannot serialize item from a null configuration section.");
         ItemMeta meta = item.getItemMeta();
+        boolean isNewVersion = XMaterial.isNewVersion();
 
-        if (meta.hasDisplayName()) config.set("name", ChatColor.stripColor(meta.getDisplayName()));
-        if (meta.hasLore()) {
-            List<String> lores = meta.getLore();
-            List<String> lines = new ArrayList<>(lores.size());
-            for (String lore : lores) lines.add(ChatColor.stripColor(lore));
-            config.set("lore", lines);
-        }
+        // Material
+        config.set("material", item.getType().name());
+
+        // Amount
         if (item.getAmount() > 1) config.set("amount", item.getAmount());
-        if (XMaterial.isNewVersion()) {
+
+        // Durability - Damage
+        if (isNewVersion) {
             if (meta instanceof Damageable) {
                 Damageable damageable = (Damageable) meta;
                 if (damageable.hasDamage()) config.set("damage", damageable.getDamage());
             }
-        } else if (item.getType().getMaxDurability() > 0) {
+        }else{
             config.set("damage", item.getDurability());
         }
-        config.set("material", item.getType().name());
+
+        // Display Name & Lore
+        if (meta.hasDisplayName())config.set("name", meta.getDisplayName());
```

Comment:
Why did you move this and removed `ChatColor.stripColor` from it?

Reply:
If the items have colored display names or lores, the method **ChatColor.stripColor** will ommit the colors.

Reply:
i moved it to organize it in a better form.

Reply:
Oh you're right, I totally forget that. I thought it's going to reverse them back to `&` good catch!